paasta_tools/cli/cmds/validate.py

```diff
@@ -661,6 +661,18 @@ def validate_min_max_instances(service_path):
             if instance_config.get_instance_type() != "tron":
```

Comment:
I'm surprised this doesn't crash on adhoc instances right now - I don't think those would have `get_min_instances`/`get_max_instances`. I'd probably change this to `== "kubernetes"`.